src/App.tsx

```diff
@@ -38,58 +39,86 @@ export function getReorderedGoods(
   // eslint-disable-next-line no-console
   console.log(sortType, isReversed);
 
+  switch (sortType) {
+    case 1:
```

Comment:
What is 1? It doesn't say anything. Also use all of the cases
```suggestion
   case SortType.ALPHABET:
```